env0/resource_template.go

```diff
@@ -410,25 +410,6 @@ func templateCreatePayloadFromParameters(prefix string, d *schema.ResourceData)
 		return payload, diag.Errorf("schema resource data serialization failed: %v", err)
 	}
 
-	isNew := d.IsNewResource()
```

Comment:
this was the code to handle only for new resources... this is the breaking change...